src/app/views/outreach/outreach.component.html

```diff
@@ -1,14 +1,14 @@
-<div class="ng-container p-5 mx-5">
-  <h1>Outreach</h1>
+<div class="ng-container text-center p-5 mb-5">
+  <h1><b>Outreach</b></h1>
 </div>
-<div class="p-5 row text-center">
-  <div class="col-md-6">
+<div class="px-5 pb-5 row text-center">
+  <div class="col-md-6 col-sm-12 ">
     <h2 >Floods in a changing climate</h2><br>
-    <iframe width="600" height="350" src="https://www.youtube.com/embed/yTR90oWjb5M" title="Floods in a Changing Climate | Dr P P Mujumdar | October 26, 2020" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+    <iframe class="youtube-player" width="100%" height="350vh" src="https://www.youtube.com/embed/yTR90oWjb5M" title="Floods in a Changing Climate | Dr P P Mujumdar | October 26, 2020" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
   </div>
-  <div class="col-md-6">
+  <div class="col-md-6 col-sm-12 ">
     <h2>Vulnerability of Water Resources to Climate Change</h2><br>
-    <iframe width="600" height="350" src="https://www.youtube.com/embed/2-tiGpm4Pao" title="iSEE Congress 2015 — Session 1: &quot;Vulnerability of Water Resources to Climate Change&quot;" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+    <iframe class="youtube-player" width="100%" height="350vh" src="https://www.youtube.com/embed/2-tiGpm4Pao" title="iSEE Congress 2015 — Session 1: &quot;Vulnerability of Water Resources to Climate Change&quot;" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

Comment:
Height: 350vh,
Please go through the meaning of vh usage once and try to understand the meaning generated here